pandas/tests/resample/test_datetime_index.py

```diff
@@ -449,9 +449,7 @@ def test_resample_frame_basic_kind(freq, unit):
         index=date_range("2000-01-01", periods=10, freq="B"),
     )
     df.index = df.index.as_unit(unit)
-    msg = "The 'kind' keyword in DataFrame.resample is deprecated"
-    with tm.assert_produces_warning(FutureWarning, match=msg):
-        df.resample(freq, kind="period").mean()
```

Comment:
i think you can remove this test entirely

Reply:
thanks, done